### PR TITLE
Updated links to static files in new EGA website

### PR DESCRIPTION
**Pull request by malloryfreeberg**
## Description

- Updated URLs to static documents in EGA website (links were broken with release of the new website in September 2023)

## Motivation and Context

This change fixes broken links to important FEGA documents.

## How has this been tested?

URLs were copied and pasted into a Chrome browser to confirm they were the correct ones to use.

## What should reviewers focus on?

Check that updated URLs load the intended documents.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New content (non-breaking change which adds new content)
- [ ] Modified content (non-breaking change which modifies existing content)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] I have read the CONTRIBUTING.md guide.
- [x] My changes follow the code style of this project.
- [x] I have updated supporting documentation, if required.
- [x] I have added my details to the CONTRIBUTORS.yaml file.

### CODE REVIEW

**Review by M-casado — CHANGES_REQUESTED** [review]
Links look good and work, but I searched all inside the repo for similar links:
````
$ grep -R "ega-archive.org/files" .
./docs/topics/data-metadata-management/index.md:Central EGA has a <a href="https://ega-archive.org/files/European_Genome_phenome_Archive_Security_Overview.pdf" target="_blank">Security Strategy</a> which defines best practices in ensuring data are stored securely. The EGA has helped develop the recommendations outlined the <a href="https://www.ga4gh.org/docs/ga4ghtoolkit/data-security/2016May10_REV_SecInfrastructure.pdf" target="_blank">GA4GH Security Technology Infrastructure document</a>, which defines guidelines, best practices, and standards for building and operating an infrastructure that promotes responsible data sharing in accordance with the <a href="https://www.ga4gh.org/docs/ga4ghtoolkit/data-security/Privacy-and-Security-Policy.pdf" target="_blank">GA4GH Privacy and Security Policy</a>.
./docs/topics/governance-legal/index.md:Here is described the <a href="https://ega-archive.org/files/EGA-Federation-Structure-v1.1.pdf" target="_blank">Federated EGA Structure and Organisation</a>. Federated EGA is broadly organised into two main types of nodes: Central EGA nodes (maintained by EMBL-EBI and CRG) and Federated EGA nodes (maintained at national or regional levels). Read the document linked above to learn more about the roles and responsibilities of each type of node.
./docs/topics/governance-legal/index.md:- The **Federated EGA Strategic Committee** <a href="https://ega-archive.org/files/ToR-Federated-EGA-Strategic-Committee-v1.1.pdf" target="_blank">(Terms of Reference)</a> provides direction and strategic planning for the Federated EGA. The Committee receives input from the Central EGA Strategic Committee and provides feedback for the EGA strategic roadmap.
./docs/topics/governance-legal/index.md:- The **Federated EGA Operations Committee** <a href="https://ega-archive.org/files/ToR-Federated-EGA-Operations-Committee-v1.1.pdf" target="_blank">(Terms of Reference)</a> reviews operational performance and coordinates technical implementation roadmaps of the Federated EGA. The Committee receives advice from and provides operational reporting to the Federated EGA Strategic Committee.
./docs/topics/governance-legal/index.md:As Central EGA is a Data Processor under GDPR definitions, all data submitters are required to agree to and sign the <a href="https://ega-archive.org/files/EGA_Data_Processing_Agreement_v1.1.pdf" target="_blank">EGA Data Processing Agreement</a>. Federated EGA nodes also acting as Data Processors will require a similar agreement outline the roles and responsibilities of Data Processors, Data Controllers, and Data Requestors.
./docs/topics/technical-operational/index.md:- The <a href="https://ega-archive.org/files/EGA-Node-Operations-v2.pdf" target="_blank">FEGA Node Operations Guidelines</a> document gives an overview of the operational areas which require resources in order to establish and operate a Federated EGA node. The document is based on more than 10 years experience of establishing and operating the EMBL-EBI and CRG Central EGA nodes. It provides a breakdown of the operational areas of responsibility into Helpdesk Services, Technical Operations, Software Development, and IT Infrastructure.
````
I found two other broken links within ``./docs/topics/data-metadata-management/index.md`` and ``./docs/topics/technical-operational/index.md`` that we can easily fix within this same PR. I also checked that adding ``assets/`` to the URL fix these as well

File: docs/topics/data-metadata-management/index.md
Comment (M-casado) [inline docs/topics/data-metadata-management/index.md:1]: Other links that I noticed broken or wrong:

- [ ] ``https://ega-archive.org/beacon/#/`` --> ``https://ega-archive.org/about/projects-and-funders/beacon/``
- [ ] ``https://ega-archive.org/ga4gh`` --> ``https://ega-archive.org/about/projects-and-funders/ga4gh/``
- [ ] ``https://ega-archive.org/beacon-api/`` --> 🤷 
- [ ] ``https://ega-archive.org/data-use-conditions`` --> ``https://ega-archive.org/access/data-access-committee/data-use-ontology/``
- [ ] ``https://ega.ebi.ac.uk:8052/elixir/tickets/tickets`` --> 🤷 
- [ ] ``https://web3.ega-archive.org/metadata/validation/metadata-schema/`` --> ``https://ega-archive.org/submission/metadata/ega-schema/``
- [ ] ``https://www.youtube.com/watch?v=josXa1CydxA`` --> 🤷

**Review by M-casado — APPROVED** [review]
Some links were fixed, but when checking the whole documents I noticed quite a few that are broken in the ``data-metadata-management`` file. 
Either in this PR or in the next one we should fix them.

File: docs/topics/data-metadata-management/index.md
Comment (malloryfreeberg) [reply docs/topics/data-metadata-management/index.md:1]: Thanks @M-casado . Can you make a new GH issue with a list of these broken links? We can do them in another branch/PR.

**Pull request merged**